addons/captives/functions/fnc_setHandcuffed.sqf

```diff
@@ -41,6 +41,11 @@ if ((_unit getVariable [QGVAR(isHandcuffed), false]) isEqualTo _state) exitWith
 if (_state) then {
     _unit setVariable [QGVAR(isHandcuffed), true, true];
     [_unit, "setCaptive", QGVAR(Handcuffed), true] call EFUNC(common,statusEffect_set);
+    call FUNC(endRadioTransmission);
```

Comment:
This should probably be `EFUNC(common,endRadioTransmission)` as it's locat.ed in `common` and you are doing this in `captives`. [This](http://ace3mod.com/wiki/development/coding-guidelines.html#macro-usage) should explain it.